.github/workflows/test-and-demo.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
       run: npm run chromatic
     - name: Extract branch name
       shell: bash
-      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/} | sed 's/[^A-Za-Z-]/-/g')"
```

Comment:
Looks good so far (I think underscores are also possible, and caps are not allowed (according to the [docs](https://wmde.github.io/wikit/?path=/story/documentation-contributing--page#branches), though I'm not sure why caps or digits are not allowed)

Reply:
BTW, If you want to validate this before merging, you want to add this particular file to the globbing pattern at the begining of this workflow, so that CI runs when this is changed.

Reply:
right, I meant to added but failed. I also meant to include digits, done in another version.

Re: validating. I actually was trying to push a noop code change on top of those to trigger github action. No success so far, mostly because of failures with amending the sed commit

Reply:
well, the workflow itself isn't triggering though, I think it might be because this workflow only runs when you push to the `vue-components` subfolder. I think if you want to see if it works IRL then you need to add `.gihub/workflows/test-and-demo.yml` to the `paths` list in line 11